athlinks-schema.ttl

```diff
@@ -158,7 +195,7 @@ athlinks:pace
 athlinks:seriesType
   a owl:DataTypeProperty;
   rdfs:range [ a xsd:string ;
-                owl:oneOf("Master" "Social" "Points" "Series")
+                owl:oneOf("Master" "Social" "Points" "Series" "MasterSeries")
```

Comment:
Maybe we could do this with some kind of a recurrence property instead of using a "type" to define an interval.